src/rgw/rgw_common.h

```diff
@@ -239,6 +239,7 @@ using ceph::crypto::MD5;
 #define ERR_POSITION_NOT_EQUAL_TO_LENGTH                 2219
 #define ERR_OBJECT_NOT_APPENDABLE                        2220
 #define ERR_INVALID_BUCKET_STATE                         2221
+#define ERR_INCOMPLETE_WRITE_REQ 2222
```

Comment:
the name of this error should mention 'continue' - something like ERR_EXPECTED_100_CONTINUE, ERR_BEFORE_100_CONTINUE, etc